### PR TITLE
Add entities to shopping list integration

### PR DESCRIPTION
**Pull request by pkuehne**
## Proposed change

This adds a reference to the entities created in the shopping list when the codebase PR is merged.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  home-assistant/core#85389
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/shopping_list.markdown [hunk 1]
```diff
@@ -11,12 +11,22 @@ ha_domain: shopping_list
 ha_integration_type: integration
 ---
 
-The `shopping_list` integration allows you to keep track of shopping list items. 
+The `shopping_list` integration allows you to keep track of shopping list items.
 
-Your shopping list will be accessible from the sidebar, and you can optionally add the [Shopping List card](/dashboards/shopping-list/) to your dashboard. With the [Conversation integration](/integrations/conversation/) you can add items to your shopping list using voice commands like "Add eggs to my shopping list." 
+Your shopping list will be accessible from the sidebar, and you can optionally add the [Shopping List card](/dashboards/shopping-list/) to your dashboard. With the [Conversation integration](/integrations/conversation/) you can add items to your shopping list using voice commands like "Add eggs to my shopping list."
```
Comment (frenck) [inline source/_integrations/shopping_list.markdown]:
```suggestion
Your shopping list will be accessible from the sidebar, and you can optionally add the [Shopping List card](/dashboards/shopping-list/) to your dashboard. With the [Conversation integration](/integrations/conversation/), you can add items to your shopping list using voice commands like "Add eggs to my shopping list."
```

File: source/_integrations/shopping_list.markdown [hunk 2]
```diff
@@ -11,12 +11,22 @@ ha_domain: shopping_list
...
 
+## Entities
+
+The integration creates three entities, which expose the counts of the number of items in the shopping list. There is one each for:
+
+* The total number of items
+* The number of incomplete items
+* The number of completed items
+
+These are useful when used in automations, for example to notify you if you're at the shops and there are incomplete items in your shopping list.
```
Comment (frenck) [inline source/_integrations/shopping_list.markdown]:
```suggestion
The integration creates three entities, which expose the counts of the number of items in the shopping list. There is one each for:

- The total number of items
- The number of incomplete items
- The number of completed items

These are useful when used in automations, for example, to notify you if you're at the shops and there are incomplete items in your shopping list.
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by pkuehne** [comment]
Thank you!

**Pull request closed without merging**